### ISSUE TITLE
训练一直卡在loading base model中, 使用deepspeed zero3.config的时候稳定复现

### ISSUE THREAD

**Issue by samaritan1998** [Yuliang-Liu/Monkey#49]
8卡a100，需要对训练脚本做什么修改么？

**Comment by samaritan1998** [comment]
8卡a100*40G

**Comment by Ryoo72** [comment]
same question.

**Comment by samaritan1998** [comment]
> same question.

请问你解决了么？

**Comment by Ryoo72** [comment]
No. I haven't solved it yet. Actually, I gave up🥲 It's difficult to use the translator. Please speak in English...

**Comment by samaritan1998** [comment]
![image](https://github.com/Yuliang-Liu/Monkey/assets/48083986/77eaeaaf-a6f8-4fd7-afc2-e5351d5e27e5)
附内存占用情况

**Comment by Ryoo72** [comment]
There are two types of a100: 40GB and 80GB. The author seems to have trained with 80GB.

**Comment by zc-zhao** [comment]
When I use zero3, there is a situation where the loss is larger than when I use zero2. It seems that the model did not initialize successfully. Have you ever encountered this when using  zero3?
![微信图片_20240314172641](https://github.com/Yuliang-Liu/Monkey/assets/92244583/3374601d-0ddb-4d68-9da4-6642501882a4)

![微信图片_20240314172647](https://github.com/Yuliang-Liu/Monkey/assets/92244583/ea762629-5df5-43f8-972a-d11944e3c118)
The first picture shows the loss of zero3 and the second picture shows the loss of zero2, and there is a parameter mismatch issue during model initialization, as follows:

![b0c10984d8765f94751df92bee2bc56](https://github.com/Yuliang-Liu/Monkey/assets/92244583/9d299f09-57fb-4523-8eae-4222a0ad4f44)

**Comment by samaritan1998** [comment]
Training has been stuck at loading the base model, and I haven't been able to successfully train yet.

**Comment by luohao123** [comment]
Does it training from scratch or loading the textmoney weights finetune?

**Comment by samaritan1998** [comment]
> Does it training from scratch or loading the textmoney weights finetune?
loading the monkey weights finetune

**Comment by luohao123** [comment]
This is not good, we need train from scratch

**Comment by MelosY** [comment]
Could you please give more detail information about your training? Our model use the Qwen-VL as pretrained model and it can work well in 8xA800 80G with ZeRO2.

**Comment by samaritan1998** [comment]
> Could you please give more detail information about your training? Our model use the Qwen-VL as pretrained model and it can work well in 8xA800 80G with ZeRO2.您能提供有关您的培训的更多详细信息吗？我们的模型使用 Qwen-VL 作为预训练模型，它可以在 8xA800 80G 和 ZeRO2 中很好地工作。

40G 8卡 A100
